<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>languageChoiceWidget</class>
 <widget class="QWidget" name="languageChoiceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="studentName">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="languageChoice">
     <item>
      <property name="text">
       <string>английский</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>немецкий</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>французский</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="languageChoiceDiploma">
     <item>
      <property name="text">
       <string>русский</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>английский</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>немецкий</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>французский</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
